cmd/manager/main.go

```diff
@@ -61,23 +63,24 @@ func main() {
 		log.Fatal(err)
 	}
 
-	status := status.New(cfg, mgr, namespace)
```

Comment:
Do you really need this variable change now that you are just adding to the older implementation? Not changing the variable name will reduce the code churn.

Reply:
I implemented the name change because the variable `status` conflicted with the package of the same name. If you don't believe this is an issue I can revert the change.

Reply:
This shouldn't be an issue given that's the way it was before. But now that you point that out I agree with you.

Reply:
It's doesn't appear to be an issue unless you want to use a different method from the `status` package - once the variable is declared with the same name as the package you will only be able to use methods from the `Status` interface which the status stuct implements.